lightstep/sdk/metric/internal/viewstate/viewstate_test.go

```diff
@@ -1231,3 +1231,196 @@ func TestDeltaTemporalityMinMaxSumCount(t *testing.T) {
 		),
 	)
 }
+
+func TestViewHints(t *testing.T) {
+	views := view.New("test")
+	vc := New(testLib, views)
+	otelErrs := test.OTelErrors()
+
+	histo, err := testCompile(
+		vc,
+		"histo",
+		sdkinstrument.SyncCounter, // counter->small histogram
+		number.Float64Kind,
+		instrument.WithDescription(`{
+  "aggregation": "histogram",
+  "config": {
+    "histogram": {
+      "max_size": 3
+    }
+  }
+}`))
+	require.NoError(t, err)
+
+	mmsc, err := testCompile(
+		vc,
+		"mmsc",
+		sdkinstrument.SyncHistogram, // histogram->minmaxsumcount
+		number.Float64Kind,
+		instrument.WithDescription(`{
+  "description": "heyyy",
+  "aggregation": "minmaxsumcount"
+}`))
```

Comment:
nit: formatting

Reply:
`go fmt` does this with back-quoted strings. 🤷‍♂️

Reply:
ugh, not sure why, but sure, okay `go fmt`

Reply:
They're multi-line strings, so white space is meaningful? I'm not sure how else to format this. Anyway, this is why I love "go fmt" :)